pkg/subctl/cmd/verify.go

```diff
@@ -64,10 +64,17 @@ func addVerifyFlags(cmd *cobra.Command) {
 var verifyCmd = &cobra.Command{
 	Use:   "verify <kubeConfig1> <kubeConfig2>",
 	Short: "Run verifications between two clusters",
-	Long: `The verify command is controlled by the --verify-only and --enable-disruptive flags,
-all verifications in --enable-only will be executed, but the disruptive ones, like gateway
-failover testing will be excluded filtered out unless --enable-disruptive is provided. If in
-interactive mode the user will be asked about running disruptive testing.`,
+	Long: `This command performs various tests to verify that a Submariner deployment between two clusters
+is functioning properly. The verifications performed are controlled by the --only and --enable-disruptive
+flags. All verifications listed in --only are performed with special handling for those deemed as disruptive.
+A disruptive verification is one that changes the state of the clusters as a side effect. If running the
+command interactively, you will be prompted for confirmation to perform disruptive verifications unless
+the --enable-disruptive flag is also specified. If running non-interactively (that is with no stdin),
+--enable-disruptive must be specified otherwise disruptive verifications are skipped.
+
+The following verifications are deemed disruptive:
+
+    gateway-failover`,
```

Comment:
Could this be constructed from `verifyE2EDisruptivePatterns`?

Reply:
yeah good point.

Reply:
Done